### ISSUE TITLE
Deprecated SafeConfigParser

### ISSUE THREAD

**Issue by xgaia** [trendmicro/flask-ini#7]
Hello,

I use FlaskIni, and when I test my project with `pytest`, I have this warning:

```python
app.py:90: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
    app.iniconfig = FlaskIni()
```

Can you fix it by using `ConfigParser` instead of `SafeConfigParser`?

**Comment by pedantic-git** [comment]
Hi @xgaia - I no longer work for Trend Micro and I haven't used Flask in years, so it would be difficult for me to test this and get it onto PyPi. If you're interested in becoming the maintainer or know someone who is, that would be great.

**Comment by xgaia** [comment]
Yes, you can transfer the repo to my github account. I don't know if its possible to transfere ownership in pypi.

**Comment by pedantic-git** [comment]
@xgaia I can't transfer it because I no longer work for Trend Micro. But please feel free to take a fork.
